src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Executor
```

Comment:
When considering a name we should probably not have a class as the same name as the namespace because when you have to write this, Executor.Executor.OpenFile(foo.txt), it feels weird